sdk/communication/Azure.Communication.CallingServer/src/Models/DtmfOptionsInternal.cs

```diff
@@ -6,10 +6,13 @@
 
 namespace Azure.Communication.CallingServer
 {
-    [CodeGenModel("DtmfConfigurations")]
-    internal partial class DtmfConfigurationsInternal
+    [CodeGenModel("DtmfOptions")]
+    public partial class DtmfOptionsInternal
     {
+        /// <summary>
+        /// Tones to stop by.
```

Comment:
Could we enhance our docs to customers? What does this mean exactly?

Reply:
Updated on my PR.